apps/src/code-studio/pd/workshop_enrollment/enroll_form.tsx

```diff
@@ -1,174 +1,193 @@
-  'Grade 2',
-  'Grade 3',
-  'Grade 4',
-  'Grade 5',
-  'Grade 6-8',
-  'Grade 9-12',
-];
-
-const CSF_COURSES = {
-  courseA: 'Course A',
-  courseB: 'Course B',
-  courseC: 'Course C',
-  courseD: 'Course D',
-  courseE: 'Course E',
-  courseF: 'Course F',
-  express: 'Express',
-  courses14_accelerated: 'Courses 1-4 or Accelerated',
-};
-
-const ATTENDED_CSF_COURSES_OPTIONS = {
-  'Yes, I attended a CS Fundamentals Intro workshop this academic year.':
-    'Yes, this year',
-  'Yes, I attended a CS Fundamentals Intro workshop in a previous academic year.':
-    'Yes, prior year',
-  'Nope, I have never attended a CS Fundamentals workshop.': 'No',
-};
+import styles from '@cdo/apps/code-studio/pd/workshop_enrollment/EnrollForm/styles.module.scss';
+import textFieldStyles from '@cdo/apps/componentLibrary/textField/textfield.module.scss';
 
 interface SchoolInfoProps {
+  country?: string;
   school_id?: string;
-  school_district_name?: string;
-  school_district_other?: string;
   school_name?: string;
-  school_state?: string;
+  school_type?: string;
   school_zip?: string;
-  school_type?: keyof typeof SCHOOL_TYPES_MAPPING;
 }
 
 interface EnrollFormState {
-  attended_csf_intro_workshop?: keyof typeof ATTENDED_CSF_COURSES_OPTIONS;
-  csf_course_experience?: Partial<typeof CSF_COURSES>;
-  csf_courses_planned?: string[];
-  csf_intro_intent?: string;
-  csf_intro_other_factors?: string;
-  describe_role?: string;
-  email?: string;
-  confirm_email?: string;
-  explain_csf_course_other?: string;
-  explain_not_teaching?: string;
-  explain_teaching_other?: string;
-  first_name?: string;
-  grades_teaching?: string[];
-  last_name?: string;
-  planning_to_teach_ap?: string;
-  previous_courses: Array<string>;
-  role?: string;
-  taught_ap_before?: string;
-  years_teaching?: string;
-  years_teaching_cs?: string;
+  attended_csf_intro_workshop: keyof typeof ATTENDED_CSF_COURSES_OPTIONS;
+  confirm_email: string;
+  csf_course_experience: Partial<typeof CSF_COURSES>;
+  csf_courses_planned: string[];
+  csf_intro_intent: string;
+  csf_intro_other_factors: string[];
+  describe_role: string;
+  email: string;
+  explain_csf_course_other: string;
+  explain_not_teaching: string;
+  explain_teaching_other: string;
+  first_name: string;
+  grades_teaching: string[];
+  last_name: string;
+  planning_to_teach_ap: string;
+  previous_courses: string[];
+  role: string;
+  taught_ap_before: string;
+  years_teaching: string;
+  years_teaching_cs: string;
 }
 
-type CombinedFormState = keyof EnrollFormState | keyof SchoolInfoProps;
+type CombinedFormState = keyof EnrollFormState | 'school_info';
 
 type FormErrors = Partial<Record<CombinedFormState, string>>;
 
 type EnrollmentResponse = {
-  workshop_enrollment_status: string;
   account_exists: boolean;
-  sign_up_url: string;
   cancel_url: string;
+  sign_up_url: string;
+  workshop_enrollment_status: string;
 };
 
-type EnrollFormProps = EnrollFormState & {
+type EnrollFormProps = {
```

Comment:
Why did so many things need to be added? Is this cleanup or a new need? (If it's cleanup, I think it makes sense in this PR, just want to understand!)

Reply:
I needed a way to render the component with some initial state in order to avoid an even more extensive refactor of the test file. the added optional props are not actually passed to the component where it's implemented in `workshop_enroll.jsx`, only in the tests

Reply:
I will add a comment explaining this

Reply:
Did you make a ticket to move the test to RTL?

Reply:
just did! https://codedotorg.atlassian.net/browse/ACQ-2917